include/nccl_ofi_deque.h

```diff
@@ -23,21 +23,20 @@ extern "C" {
  * this structure as a black box. Critically, the caller must ensure the
```

Comment:
My comment in https://github.com/aws/aws-ofi-nccl/pull/521/commits/52a6994c1f2ce34092f720b3c7c1d6d39a067f80#r1714461159 needs to be addressed.